hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/rollback/BaseRollbackHelper.java

```diff
@@ -197,14 +197,21 @@ protected List<HoodieRollbackStat> deleteFiles(HoodieTableMetaClient metaClient,
             // if first rollback attempt failed and retried again, chances that some files are already deleted.
             isDeleted = true;
           }
+
+          if (!isDeleted) {
```

Comment:
In which case the `metaClient.getFs().delete()` can return false if the file actually exists there?

Reply:
https://hadoop.apache.org/docs/r2.6.1/api/org/apache/hadoop/fs/FileSystem.html#delete(org.apache.hadoop.fs.Path,%20boolean)